test/test_nn.py

```diff
@@ -2505,6 +2526,131 @@ def test_embedding_bag(self):
         self._test_EmbeddingBag(False, 'sum', True)
         self._test_EmbeddingBag(False, 'mean', True)
 
+    @staticmethod
+    def _get_reduction(mode='mean', dim=0):
```

Comment:
I don't understand the purpose of this function

Reply:
It took me a while to remember why I wrote this. It turns out that we don't actually need it